features.org

```diff
@@ -89,113 +57,134 @@ List of [[https://orgmode.org/worg/org-contrib/babel/languages.html][supported l
+Org includes a powerful table editor.
 
-All done with just plain text markup.
+=tab= moves between columns. =return= moves between rows.
+Layout updates automatically as cell contents change.
 
-Switching from one TODO keyword to another is =C-c C-t= or
-=S-<left/right>=.
-Want a different set of keywords to track your task?
-No problem --- Org lets you customize them to support your workflow.
+Import tables from =.csv= and =.tsv= files, or directly from the
+contents of the current buffer, then export them to any format
+supported by Org.
 
-Each item can also receive planning meta-data: scheduled and/or deadline
-cookies, tags, properties, etc.
+Org tables aren't just static views of data.  They can be used as a
+full [[https://orgmode.org/manual/The-Spreadsheet.html][spreadsheet]]
+system, with formulas, references, and more -- all with the simplicity
+and transparency of plain text.
 
 *** More
 
-More on [[file:manual/TODO-Items.html][TODO items]] (manual)
+The [[file:manual/Built_002din-Table-Editor.html][built-in table editor]] (manual)
 
-Handling [[file:manual/Dates-and-Times.html][dates and times]] (manual)
+Org as a [[file:manual/The-Spreadsheet.html][spreadsheet]] system (manual)
+
+Introduction to [[https://orgmode.org/worg/org-tutorials/tables.html][editing tables]] (Worg)
 
-* Tables
+Using [[https://orgmode.org/worg/org-tutorials/org-spreadsheet-intro.html][Org as a spreadsheet]] (Worg)
+* Superior Source Code
   :PROPERTIES:
-  :CUSTOM_ID: tables
+  :CUSTOM_ID: babel
   :HTML_CONTAINER_CLASS: side-fig
   :END:
 
-#+attr_html: :class main-image :alt Manipulating tables with Org, moving cells/rows and executing formulas
-[[file:resources/img/features/tables.gif]]
+#+attr_html: :class main-image :alt Demonstration of Org Babel executing code blocks
+[[file:resources/img/features/babel.gif]]
 
 ** Description
 
-Org is a great plain text table editor.
-
-Use =TAB= to move between cells, and =ENTER= for rows. The layout is automatically
-reformatted as cell contents change.
-
-You can import tables from =.csv= and =.tsv= files, or directly from the
-current buffer; you can then export them in all the formats supported by
-Org (see below).
-
-Org can also be used as a /spreadsheet/ system.
-
-You can add formulas using the [[http://www.gnu.org/software/emacs/manual/html_mono/calc.html][Calc Emacs package]] or Emacs lisp code.
+Org provides functionality far beyond that of computational notebooks
+such as [[https://jupyter.org/][Jupyter]] or
+[[https://rmarkdown.rstudio.com/][R Markdown]].
+
+# Program your way.
+[[https://orgmode.org/worg/org-contrib/babel/][Org babel]]
+transforms the lowly executable source block into a complete
+[[https://en.wikipedia.org/wiki/Literate_programming][literate programming]]
+environment, allowing you to weave the inputs and outputs of multiple
+languages and runtime environments across multiple machines into a
+single unified system, empowering you to achieve new levels of DRYness
```

Comment:
Maybe have "DRYness" link to https://en.wikipedia.org/wiki/Don%27t_repeat_yourself or similar?

Reply:
Good idea.